source/_integrations/switch.command_line.markdown

```diff
@@ -93,13 +93,13 @@ switch:
       command_off: "/usr/bin/curl -X GET http://192.168.1.10/digital/4/0"
       command_state: "/usr/bin/curl -X GET http://192.168.1.10/digital/4"
       value_template: '{% raw %}{{ value == "1" }}{% endraw %}'
-      friendly_name: Kitchen Lightswitch
+      friendly_name: Kitchen Light
```

Comment:
This makes it more confusing IMHO since we are pulling in a switch, not a light.